Sekoia.io/sekoiaio/triggers/base.py

```diff
@@ -163,8 +164,20 @@ def handler_dispatcher(self, raw_message: str):
             return
 
         # We can only manage v2 events
-        if str(message.get("metadata", {}).get("version")) != "2":
-            self.log("Received event with version not handled by the trigger", level="info", event=message)
+        event_version = message.get("metadata", {}).get("version")
+        if event_version is None:
+            # Bypass the version check for backward compatibility
+            # Looks like in some cases version might be missing.
+            self.log(
+                "Received event without version specified. Default version is `2`.", level="warning", event=message
+            )
+        elif str(event_version) != "2":
+            self.log(
+                f"Received event with version that cannot be handled by the trigger."
+                + f" Supported version is `2`, but received version is `{event_version}`",
```

Comment:
Just to be consistent with the previous comment
```suggestion
                + f" Expected version is `2`, but received version is `{event_version}`",
```